.goreleaser.yml

```diff
@@ -48,15 +48,14 @@ archives:
       {{- if (eq . "6") -}}hf
       {{- else -}}v{{- . -}}
       {{- end -}}
-    {{- end -}}
```

Comment:
Should we preserve this? I'm not 100% sure what should be the right syntax here.

Reply:
I added this `{{- end -}}` at the end of line 53. It doesn't throw any error so I guess it's the accepted syntax. But I can add `{{- end -}}` to the new line (just like the old syntax) just in case. Should I do that?

Reply:
@realshuting

Reply:
This workflow runs when publishing a release, it doesn't run upon every push.

Kindly ping @eddycharly to review it when he's back.

Reply:
The pattern looks wrong to me:
- aren't we missing an `{{ end }}` ?
- we already have `{{ .Arch -}}` in the first line of the template

Reply:
Maybe something like this ?
```
kyverno-cli_{{ .Tag }}_{{ .Os }}_
{{- with .Arch -}}
  {{- if eq . "amd64" -}}x86_64{{- else if eq . "386" -}}i386{{- else -}}{{- . -}}{{- end -}}
{{- end -}}
{{- with .Arm -}}
  {{- if eq . "6" -}}hf{{- else -}}v{{- . -}}{{- end -}}
{{- end -}}
```

Reply:
On the other hand we're not building for `386` and only `arm64` 🤷

Reply:
Should this:
```yml
kyverno-cli_{{ .Tag }}_{{ .Os }}_{{ .Arch -}}
    {{- with .Arm -}}
      {{- if (eq . "6") -}}hf
      {{- else -}}v{{- . -}}
      {{- end -}}
    {{- if eq .Arch "amd64" }}x86_64
    {{- else if eq .Arch "386" }}i386
    {{- else }}{{ .Arch }}{{ end }}
```

Be replaced by :
```yml
kyverno-cli_{{ .Tag }}_{{ .Os }}_
{{- with .Arch -}}
  {{- if eq . "amd64" -}}x86_64{{- else if eq . "386" -}}i386{{- else -}}{{- . -}}{{- end -}}
{{- end -}}
{{- with .Arm -}}
  {{- if eq . "6" -}}hf{{- else -}}v{{- . -}}{{- end -}}
{{- end -}}
```

Also, `{{- with .Arch -}}` shows a syntax error.

Reply:
> Also, {{- with .Arch -}} shows a syntax error.

What do you mean ?